cmd/hyperkube/kube-apiserver.go

```diff
@@ -27,8 +27,10 @@ func NewKubeAPIServer() *Server {
 	s := options.NewServerRunOptions()
 
 	hks := Server{
-		SimpleUsage: "apiserver",
-		Long:        "The main API entrypoint and interface to the storage system.  The API server is also the focal point for all authorization decisions.",
+		name:            "apiserver",
```

Comment:
It's not 100% required, but I think it's a little bit cleaner than just relying on SimpleUsage for this